lib/routes/bookings/user/tests/routes.test.js

```diff
@@ -1,10 +1,20 @@
 import { resetAndMockDB } from 'utils/testUtils';
-import { mockData } from 'utils/mockData';
+import { mockData, createMockTokenWithScope } from 'utils/mockData';
+import { SCOPE_TYPE, OAUTH_CLIENT_ID } from 'utils/constants';
 
 const { MOCK_BOOKING: bookings } = mockData;
 
-describe('/user route tests ', () => {
+describe('/user route tests', () => {
     let server;
+    const superAdminToken = createMockTokenWithScope(
```

Comment:
We could add tests to check if invalid scopes are rejected

Reply:
Hmm interesting, we could do that.